### PR TITLE
[stable25] Add documentation for iMIP API

### LINKED ISSUE #9247: Dev docs for ``\OCP\Calendar\ICreateFromString::handleIMipMessage``
Added in https://github.com/nextcloud/server/pull/33001

### PR DESCRIPTION
**Pull request by miaulalala**
Fixes #9247

image

### CODE REVIEW

File: developer_manual/digging_deeper/groupware/calendar.rst [hunk 1]
```diff
@@ -161,6 +161,29 @@ Calendars that only return `ICalendar` are implicitly read-only. If your app's c
 
     }
 
+Handling iMIP data 
+~~~~~~~~~~~~~
```
Comment (ChristophWurst) [inline developer_manual/digging_deeper/groupware/calendar.rst:165]: underline has to be as long or longer than the heading

**Comment by ChristophWurst** [comment]
> Fixes #9247

I'd say there are two aspects of it

1) Apps that provide a calendar
2) Apps that use the provided calendars

Right now it covers 1), although it's a *must*, not *may* in the current API. *Every* writable calendar has to handle iMip.

**Comment by ChristophWurst** [comment]
- [ ] Todo: also note the new method at https://docs.nextcloud.com/server/latest/developer_manual/app_publishing_maintenance/app_upgrade_guide/upgrade_to_25.html#back-end-changes should there be an app that already implemented ICreateFromString (breaking change)

**Comment by miaulalala** [comment]
> * [ ]  Todo: also note the new method at https://docs.nextcloud.com/server/latest/developer_manual/app_publishing_maintenance/app_upgrade_guide/upgrade_to_25.html#back-end-changes should there be an app that already implemented ICreateFromString (breaking change)

Added to the collective ticket at https://github.com/nextcloud/server/issues/32117

**Comment by miaulalala** [comment]
this needs to go into 25 only.

**Pull request merged**